### DISCUSSION TITLE
Expose the releaseTimestamp as a template field

### DISCUSSION THREAD

**Discussion by michaeldop** [renovatebot/renovate#22796] [Suggest an Idea]
### Tell us more.

This would allow me to update the prBodyColumns and include a age badge using something like [shields.io](https://shields.io/).

https://img.shields.io/date/1686836268.svg

**Comment by rarkins** [comment]
`releaseTimestamp` is in a string/ISO format though, looks like you'd need it as an epoch value?

**Reply by michaeldop** [reply]
Yep I just realized that. I am going to see if I can open a PR or request a feature on that project to be less stringent on time formats that it accepts.
